MAINTAINERS_GUIDE.md

```diff
@@ -29,7 +29,7 @@ It is every maintainer's responsibility to:
 * 4) Make sure their component respects the philosophy, design and
   roadmap of the project.
 
-## How are decisions made?
+## The method of making decisions
```

Comment:
How about  "How decisions are made" ?

I'm not a native English speaker, but to be honest, I prefer the original expression which looks more vivid.